corehq/sql_db/tests/utils.py

```diff
@@ -33,6 +39,22 @@ def __exit__(self, exc_type, exc_val, exc_tb):
             self.root_engine.dispose()
 
 
+def new_id_in_different_dbalias(partition_value):
```

Comment:
Just for my understanding, what is the use-case that it would be used in?

Reply:
It was added for this test

https://github.com/dimagi/commcare-hq/blob/b3c3124a10215be14b6bee6ebf3e3ca8b2fb540b/corehq/form_processor/tests/test_cases.py#L104-L120